src/table/stories/app-expansion-table.component.ts

```diff
@@ -40,6 +40,10 @@ class CustomHeaderItem extends TableHeaderItem {
 			[model]="model"
 			[size]="size"
 			[showSelectionColumn]="showSelectionColumn"
+<<<<<<< HEAD
+			[stickyHeader]="stickyHeader"
+=======
+>>>>>>> 28598a2a185bd578990271ea080d2250f321ed33
```

Comment:
I don't think you meant to commit this ...

Reply:
.........I was just testing you 🙃

Reply:
😂